uint8 frontLeft
uint8 frontRight
uint8 rearLeft
uint8 rearRight
uint8 speedFL
uint8 speedFR
uint8 speedRL
uint8 speedRR 

